djangocms_googlemap/__init__.py

```diff
@@ -1,3 +1,9 @@
-# -*- coding: utf-8 -*-
+"""
+Google Maps plugin for django CMS
+"""
 
+__author__ = 'Divio AG',
```

Comment:
I would add then in the ``setup.py`` rather then here, similar to: https://github.com/divio/djangocms-audio/blob/master/setup.py